ui/desktop/electron-app/src/ipc/handlers.js

```diff
@@ -117,6 +117,15 @@ handle('closeWindow', () => app.quit());
  */
 handle('checkCommand', async (command) => which(command, { nothrow: true }));
 
+/**
```

Comment:
All the changes on this file, except this code block has been added automatically for linting tools.